cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -148,6 +220,16 @@ OPTIONS "{" btree-option: btree-value[, ...] "}"
 a|
 Node key and uniqueness constraints with b-tree options are deprecated and need to be migrated before upgrading to 5.0.
 
+Replaced by:
+[source, cypher, role="noheader"]
+----
+CREATE CONSTRAINT
+...
+OPTIONS "{" range-option: range-value[, ...] "}"
+----
+Constraints used for string properties will also require an additional text index to cover the string queries properly.
```

Comment:
Do we want to add something around this down by the examples for text indexes? Or are we happy with the the PR now?

Reply:
I don't know how much information we had about lucene+native provider before?, the text indexes are just a kind of replacement for that.
Maybe we could add a sentence in same area as 

> Additionally, a text index is a form of single-property index, with the limitation that it only recognizes properties with string values.

hm or maybe just amend that to '... a text index is a form of single-property index that can be used to speed up string queries, ...'?
What do you think?, I don't really have the big picture of what information we have about indexes since before, can be okay to just keep current state of PR

Reply:
we didn't have that much (if any) in our part of the docs at least (don't know if planer had more in their parts)

If we change that sentence for text, how do we want to update it when adding point to the same paragraph? I think we just leave it as it is if neither of us feels strongly about updating it

Reply:
Ah yes, I was looking at the text index commit. Ok